features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -33,6 +33,14 @@ const ImportExplanationExpandable = ({
         explore: "explore",
     };
 
+    const [fileUploadActive, setFileUploadActive] = useState(false);
```

Comment:
There's already a loading condition that shows the overlay in `import.jsx`, there it is `files === null`. I personally prefer that over a flag, but what I definitely don't like is to have **two different conditions** :D